src/scss/_nav.scss

```diff
@@ -16,6 +16,7 @@
 
 	@include oGridRespondTo(M) {
 		position: relative;
+		left: 20px;
```

Comment:
ok cool. I was probably looking at the compiled source when I saw the values being written. All good then